docs/src/main/sphinx/sql/delete.md

```diff
@@ -32,6 +32,12 @@ Delete all orders:
 DELETE FROM orders;
 ```
 
+Delete all orders in the `audit` branch:
+
+```sql
+DELETE FROM orders @ audit;
+```
```

Comment:
Same question as above about showing @ within single quotes  if that is required.